README.rst

```diff
@@ -47,9 +47,9 @@ Here is a brief list of differences between ``blue`` and ``black``:
 * ``blue`` defaults to single-quoted strings.  This is probably the most
   painful ``black`` choice to our eyes, and the thing that inspired ``blue``.
   We strongly prefer using single quoted strings over double quoted strings
-  for everything *except* docstrings.  Don't ask us why we prefer double
-  quoted strings for docstrings; it just looks better to us!  For all other
-  strings, ``blue`` defaults to single quoted strings.
+  for everything *except* docstrings (and triple quoted strings).  Don't ask
```

Comment:
Can you add the "TQS" abbreviation after "and triple quoted strings"? I didn't recognize it the first time I saw it (even in context).

Reply:
Done!